src/doc/reference/glossary/cilk.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Cilk
+tags: Cilk
+---
+A simple set of extensions to the C and C++
```

Comment:
I think it's better to say, "An extension to the C and C++ programming languages..."  (It's more succinct, at least.)